tests/test_schema.py

```diff
@@ -1991,6 +2023,116 @@ def test_filter_registry_is_locked(self):
         self.assertEqual(rol2['start'], '1900-01-01T00:00:00+00:00')
         self.assertEqual(rol2['end'], '2100-01-01T00:00:00+00:00')
 
+    def test_filter_registry_is_bot(self):
+        """Check whether it returns the uuid searched when using isBot filter"""
+
+        cn = Country.objects.create(code='US',
+                                    name='United States of America',
+                                    alpha3='USA')
+
+        org_ex = Organization.objects.create(name='Example')
+        org_bit = Organization.objects.create(name='Bitergia')
+
+        indv = Individual.objects.create(mk='a9b403e150dd4af8953a52a4bb841051e4b705d9',
+                                         is_locked=True)
+        Profile.objects.create(name=None,
+                               email='jsmith@example.com',
+                               is_bot=True,
+                               gender='M',
+                               country=cn,
+                               individual=indv)
+        Identity.objects.create(uuid='A001',
+                                name='John Smith',
+                                email='jsmith@example.com',
+                                username='jsmith',
+                                source='scm',
+                                individual=indv)
+        Identity.objects.create(uuid='A002',
+                                name=None,
+                                email='jsmith@bitergia.com',
+                                username=None,
+                                source='scm',
+                                individual=indv)
+        Identity.objects.create(uuid='A003',
+                                name=None,
+                                email='jsmith@bitergia.com',
+                                username=None,
+                                source='mls',
+                                individual=indv)
+        Enrollment.objects.create(individual=indv, organization=org_ex)
+        Enrollment.objects.create(individual=indv, organization=org_bit,
+                                  start=datetime.datetime(1999, 1, 1, 0, 0, 0,
+                                                          tzinfo=dateutil.tz.tzutc()),
+                                  end=datetime.datetime(2000, 1, 1, 0, 0, 0,
+                                                        tzinfo=dateutil.tz.tzutc()))
```

Comment:
We can remove some data that is not really needed on the test to make it easier to read.